hexadecimal/hex.py

```diff
@@ -0,0 +1,87 @@
+import sys
```

Comment:
Your pull request HEX-1022 Added hex.py/nThe prefix of your message must contain prefix in ['LEETCODE', 'GENERATOR', 'TRIANGLE', 'HEXNUMBER', 'REQUESTS', 'ITERATOR']

Your commit: HEX-1022 Added hex.py
The prefix of your message must contain prefix in ['LEETCODE', 'GENERATOR', 'TRIANGLE', 'HEXNUMBER', 'REQUESTS', 'ITERATOR']
Your commit: HEX-1022 Added hex_tests.py
The prefix of your message must contain prefix in ['LEETCODE', 'GENERATOR', 'TRIANGLE', 'HEXNUMBER', 'REQUESTS', 'ITERATOR']
Your commit: HEX-1022 Added TestHexNUmber
The prefix of your message must contain prefix in ['LEETCODE', 'GENERATOR', 'TRIANGLE', 'HEXNUMBER', 'REQUESTS', 'ITERATOR']
Your commit: HEX-1022 Refactored TestHexNUmber
The prefix of your message must contain prefix in ['LEETCODE', 'GENERATOR', 'TRIANGLE', 'HEXNUMBER', 'REQUESTS', 'ITERATOR']
Your commit: HEX-1022 Added TestSolution
The prefix of your message must contain prefix in ['LEETCODE', 'GENERATOR', 'TRIANGLE', 'HEXNUMBER', 'REQUESTS', 'ITERATOR']
Your commit: HEX-1022 Refactored hex.py
The prefix of your message must contain prefix in ['LEETCODE', 'GENERATOR', 'TRIANGLE', 'HEXNUMBER', 'REQUESTS', 'ITERATOR']
Your commit: HEX-1022 Added some test in TestSolution
The prefix of your message must contain prefix in ['LEETCODE', 'GENERATOR', 'TRIANGLE', 'HEXNUMBER', 'REQUESTS', 'ITERATOR']
Your commit: HEX-1022 Refactored code
The prefix of your message must contain prefix in ['LEETCODE', 'GENERATOR', 'TRIANGLE', 'HEXNUMBER', 'REQUESTS', 'ITERATOR']